### ISSUE TITLE
[CoE Starter Kit - BUG] Stuck on Centre of Excellence Setup Wizard - Environment Request management's Get started screen

### ISSUE THREAD

**Issue by GarryPope** [microsoft/coe-starter-kit#8209]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Good morning CoE Team,

I'm going the the documentation with the step "[set up environment request management](https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-environment-components)".

I'm at the "Get started" step and receive the below pop up.

![image](https://github.com/microsoft/coe-starter-kit/assets/68440710/46f4ec14-2a63-45af-b03f-708ab76547de)

But no matter how many times I click on the X in the pop up, I cannot remove that window. I've refreshed screen, cleared cache, logged out and in, I'm always stuck there. It's saying because I didn't complete the "Initial Setup" I need to complete that first. But, I has an issue at the "initial setup" phase. where at step "Run setup flows" either the "Refresh" button never enables, or I click it and the Flows don't enabled. I had this issue twice on an install, so bypassed it and did the steps manually.

So I'm going to do the "[set up environment request components manually](https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-environment-components#set-up-the-environment-request-components-manually)" and bypass the wizard. 

Just wanted to log this as I know you wanted feedback on it. 

Thanks very much,

Garry

### Expected Behavior

When using the "Environment Request management" process in the "CoE Setup and Upgrade Wizard" app, I'm able to bypass the popup informing me I haven't completed the initial setup steps, as I've already done this manually. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.29

### What app or flow are you having the issue with?

CoE Setup and Upgrade Wizard

### What method are you using to get inventory and telemetry?

None

### Steps To Reproduce

1. Open CoE Setup and Upgrade Wizard app.
2. Don't complete the "Initial Setup".
3. Click "... More features"
4. Click "Environment Request Management"
5. When pop up appears, click X, and the popup never goes away.

### Anything else?

Note: you may not want people to skip steps in the app, which makes total sense as there must be an order to the process, but as I did the initial setup manually, and couldn't proceed with it, I wanted to come back to the app and try and did the environment set up. Thanks!

AB#3141

**Comment by Jenefer-Monroe** [comment]
This issue should be addressed in the April release with https://github.com/microsoft/coe-starter-kit/issues/7806. 
You may have unmanaged layers on the custom page? 

Here is the workaround described in that bug: https://github.com/microsoft/coe-starter-kit/issues/7806#issuecomment-1998401773

**Comment by Jenefer-Monroe** [comment]
Oh perhaps I had misunderstood then. Had the Driver flow from Core finished running when you went here? Note that it needs to complete before you do these as they list the envts in the tenant for you, so that you can exclude them from the processes if you wish.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by GarryPope** [comment]
Hello @Jenefer-Monroe,

Apologise for not replying sooner. 

Unfortunately I can't remember if the "Driver" flow has finished, but if I do another implementation, I'll keep my eye on it, and if I get the same problem, and the Driver flow has completed, I'll let you know.

Thanks very much,

Garry
